README.md

```diff
@@ -2,23 +2,23 @@
+| Aussagenlogik/Semantik/Wahrheitstabellen/TruthTableFillGaps         |   x    |  x   | `Logic.Semantics.FillGaps`                                                                                  | [`LogicTasks.Semantics.Fill`](src/LogicTasks/Semantics/Fill.hs)                                                                                            |
+| Aussagenlogik/Semantik/Wahrheitstabellen/TruthTableChooseForFormula |   x    |  x   | `Logic.Semantics.ChooseTable`                                                                               | [`LogicTasks.Semantics.Pick`](src/LogicTasks/Semantics/Pick.hs)                                                                                            |
+| Aussagenlogik/Semantik/Wahrheitstabellen/TruthTableFindMistakes     |   x    |  x   | `Logic.Semantics.FindMistakes`                                                                              | [`LogicTasks.Semantics.Decide`](src/LogicTasks/Semantics/Decide.hs)                                                                                        |
+| Aussagenlogik/Semantik/Wahrheitstabellen/TruthTableMaxterm          |   x    |  x   | `Logic.Semantics.MaxTerm`                                                                                   | [`LogicTasks.Semantics.Max`](src/LogicTasks/Semantics/Max.hs)                                                                                              |
+| Aussagenlogik/Semantik/Wahrheitstabellen/TruthTableMinTerm          |   x    |  x   | `Logic.Semantics.MinTerm`                                                                                   | [`LogicTasks.Semantics.Min`](src/LogicTasks/Semantics/Min.hs)                                                                                              |
+| Aussagenlogik/Semantik/Resolution/LogicResolutionStep               |   x    |  x   | `Logic.Semantics.ResolutionStep`                                                                            | [`LogicTasks.Semantics.Step`](src/LogicTasks/Semantics/Step.hs)                                                                                            |
+| Aussagenlogik/Semantik/Resolution/LogicResolutionComplete           |   x    |  x   | `Logic.Semantics.ResolutionFull`                                                                            | [`LogicTasks.Semantics.Resolve`](src/LogicTasks/Semantics/Resolve.hs)                                                                                      |
+| Aussagenlogik/Semantik/Resolution/PrologResolutionStep              |   x    |  x   | `Logic.Semantics.ResolutionStepProlog`                                                                      | [`LogicTasks.Semantics.Prolog`](src/LogicTasks/Semantics/Prolog.hs)                                                                                        |
```

Comment:
Die Motivation für die vielen neuen Leerzeichen ist mir nicht klar.

In die Tabelle sollte dann natürlich auch eine neue Zeile für den neuen Aufgabentyp.

Und vielleicht ist das eine gute Gelegenheit, die Tabelle um eine Spalte zu ergänzen, in der festgehalten ist, ob der jeweilige Aufgabentyp Teilpunktebewertung unterstützt.

Reply:
> Die Motivation für die vielen neuen Leerzeichen ist mir nicht klar.

Das ist wohl auf die Formatierung mit Prettier zurückzuführen. 

> In die Tabelle sollte dann natürlich auch eine neue Zeile für den neuen Aufgabentyp.

Wollen wir das nicht an die tatsächliche Einbindung im Autotool knüpfen?

> Und vielleicht ist das eine gute Gelegenheit, die Tabelle um eine Spalte zu ergänzen, in der festgehalten ist, ob der jeweilige Aufgabentyp Teilpunktebewertung unterstützt.

Gute Idee.

Reply:
> Wollen wir das nicht an die tatsächliche Einbindung im Autotool knüpfen?

Na ja, der Eintrag in der entsprechenden Tabellenspalte kann für die neue Zeile ja dann erst mal leer bleiben, was zugleich ein guter Reminder dafür wäre, dass auf Autotool-Seite ein ToDo der Anbindung besteht.

Reply:
Ich habe ein entsprechendes Issue im Autotool Repository angelegt.